src/Dialect/Mlir/CMakeLists.txt

```diff
@@ -0,0 +1,22 @@
+# SPDX-License-Identifier: Apache-2.0
+
+if (${CMAKE_SYSTEM_PROCESSOR} STREQUAL "s390x")
+  # currently needed for a bug on Z for the ONNXOps.cpp.inc file
+  llvm_replace_compiler_option(CMAKE_CXX_FLAGS_RELEASE "-O2" "-O1")
+  llvm_replace_compiler_option(CMAKE_CXX_FLAGS_RELEASE "-O3" "-O1")
+endif()
+
```

Comment:
Nit: no need for multiple new lines in a row

Reply:
ended up removing them, as the comment indicates that it is for a file that was in the old dir